integration/deployment/default.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: danm.k8s.io/v1
+kind: DanmNet
+metadata:
+  name: default
+  namespace: kube-system
+spec:
+  NetworkID: default
+  NetworkType: flannel
+---
+apiVersion: danm.k8s.io/v1
```

Comment:
we only need the default networks in the two namespaces, no need for the explicit Flannel network definition

Reply:
The current svcwatcher_ds explicitly requires flannel network.

Reply:
the accurate statement would be "the current svcwatcher requires the presence of A network connection through which it can reach the K8s API server, and because Levo uses Flannel in its Nokia internal product for the purpose, it was accidentally left in its example manifest file" :)